packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCard.tsx

```diff
@@ -44,6 +45,7 @@ export type FacetPlotsCardProps = {
   description?: string
   facetsToPlot?: string[]
   detailsPagePath?: string
+  plotType?: PlotType
```

Comment:
PlotType is something that we define, that translates into changes to the Plotly js layout and data transform